k8s/aeraki.yaml

```diff
@@ -118,7 +118,7 @@ kind: Role
 metadata:
   labels:
     app: aeraki
-  name: aeraki
+  name: aeraki-${AERAKI_NAMESPACE}
```

Comment:
Do we have to change the name?

Reply:
It is not necessary, but after installing multiple aeraki on the same cluster, this should be modified

Reply:
It's ok to use the same name for multiple Aeraki deployments since they're in different namespaces.

Reply:
Installing aeraki in different namespace, aeraki failed to start.
Refer to the istio document, it is recommended to different clusterrole and clusterrolebinding.